File: RxHttp/src/main/java/com/github/kongpf8848/rxhttp/request/DeleteRequest.java
package com.github.kongpf8848.rxhttp.request;

import android.app.Activity;
import android.content.Context;

import androidx.fragment.app.Fragment;

import okhttp3.RequestBody;

public class DeleteRequest extends AbsRequest {


    public DeleteRequest(Context context) {
        super(context);
    }

    public DeleteRequest(Fragment fragment) {
        super(fragment);
    }

    public DeleteRequest(Activity activity) {
        super(activity);
    }

    @Override
    protected RequestBody buildRequestBody() {
        return null;
    }
}

File: RxHttp/src/main/java/com/github/kongpf8848/rxhttp/request/DeleteRequest.kt
package com.github.kongpf8848.rxhttp.request

import android.app.Activity
import android.content.Context
import androidx.fragment.app.Fragment
import okhttp3.RequestBody

class DeleteRequest : AbsRequest {
    constructor(context: Context) : super(context)
    constructor(fragment: Fragment) : super(fragment)
    constructor(activity: Activity) : super(activity)

    override fun buildRequestBody(): RequestBody? {
        return null
    }
}